explainer.md

```diff
@@ -0,0 +1,55 @@
+# Audio Output Devices API explained
+
+The Audio Output Devices API allows JavaScript applications to direct the audio
+output of a media element to authorized devices other than the system or user
+agent default. This can be helpful in a variety of real-time communication
+scenarios as well as general media applications. For example, an application
+can use this API to programmatically direct output to a device such as a
+Bluetooth headset or speakerphone.
+
+The Audio Output Devices API adds a setSinkId method and a sinkId field to
+HTML media elements. The setSinkId() method allows changing the audio output
+device on which the element renders audio, and the sinkId attribute is a string
+that contains the ID of the device where the element is currently rendering
+audio.
+
+By default, the sinkId field is the empty string, which means that audio will be
+rendered on the default audio output device in the system.
+
+The set of valid device IDs that can be used as argument to setSinkId() can be
+obtained from a call to [enumerateDevices](https://w3c.github.io/mediacapture-main/#dom-mediadevices-enumeratedevices).
```

Comment:
Maybe mention that permissions for non-default audio output devices are linked to `getUserMedia` permissions

Reply:
done